css/about.css

```diff
@@ -0,0 +1,698 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap');
+
+:root {
+  --black: #272a31;
+  --orange: #ec5242;
+  --grey: #d3d3d3;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Lato', sans-serif;
+  text-transform: capitalize;
+  text-decoration: none;
+  outline: none;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+html {
+  width: 100%;
+  scroll-behavior: smooth;
+}
+
+/* -----------------------------Hamburgar Icon Starts------------------------------------------------------ */
+.hamburger {
+  position: absolute;
+  cursor: pointer;
+  width: 20px;
```

Comment:
- [ ] **[OPTIONAL]:** I noticed that you repeated some sections in both stylesheets. I would suggest not repeating yourself, in this case, you could create a separate CSS file and call it -for example- `shared.css` and put all of the shared styles such as the navbar, hamburger, etc there. Then, reference it in each HTML file. 🤓 🛠️